src/test/ui/rfcs/rfc-2396-target_feature-11/safe-calls.rs

```diff
@@ -0,0 +1,80 @@
+#![feature(target_feature_11)]
+
+#[target_feature(enable = "sse2")]
+fn sse2() {}
+
+#[target_feature(enable = "avx")]
+#[target_feature(enable = "bmi2")]
+fn avx_bmi2() {}
+
+trait FooBar {
+    fn sse2(&self);
+}
+
+struct Quux;
+
+impl Quux {
+    #[target_feature(enable = "avx")]
+    #[target_feature(enable = "bmi2")]
+    fn avx_bmi2(&self) {}
+}
+
+impl FooBar for Quux {
+    #[target_feature(enable = "sse2")]
```

Comment:
According to the RFC this should be an error.

Reply:
Oops! I missed this part of the RFC...